### ISSUE TITLE
What does `%7` mean in the JazzCorpus?

### ISSUE THREAD

**Issue by rubencart** [markgw/jazzparser#2]
Am I correct to conclude from these snippets that also in the [JazzCorpus](https://jazzparser.granroth-wilding.co.uk/JazzCorpus.html) used in [your paper](https://homepages.inf.ed.ac.uk/steedman/papers/music/jnmr2014.pdf) `%7` chords denote half-diminished 7th chords?

https://github.com/markgw/jazzparser/blob/d5cd076e25e26fa67a8769a9b31ea0bcadd7d458/src/jazzparser/misc/chordlabel/chord_vocabs.py#L190-L209

https://github.com/markgw/jazzparser/blob/d5cd076e25e26fa67a8769a9b31ea0bcadd7d458/src/jazzparser/data/__init__.py#L56-L68

**Comment by markgw** [comment]
Yes, at least as far as I can remember that's exactly what it means.
The '%7' is used as an ascii approximation to ø7.

**Comment by rubencart** [comment]
Thanks!
